addons/binding/org.openhab.binding.lgwebos/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lgwebos" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="WebOSTV">
+		<label>WebOS TV</label>
+		<description>WebOS based smart TV</description>
+
+		<channels>
+			<channel id="power" typeId="commandType">
+				<label>Power</label>
+				<description>Via this binding TV can only be powered off, not on.</description>
+			</channel>
+			<channel id="mute" typeId="commandType">
+				<label>Mute</label>
+				<description>Current Mute Setting</description>
+			</channel>
+			<channel id="volume" typeId="volumeChannelType" />
+			<channel id="channel" typeId="stringType">
```

Comment:
if it is a number, couldn't this be a Number channel instead? Then you could get rid off channelUp and channelDown as the number could be easily increased or decreased (e.g. through a setpoint widget).

Reply:
will do. It is a string in connectsdk. on webos it is a parsable to a number.